src/cpu-kernels/operations.cpp

```diff
@@ -3,7 +3,9 @@
 #include <cstring>
 
 #include "awkward/cpu-kernels/operations.h"
+class CPUKernels {
 
+};
```

Comment:
You're going with a namespace now, so don't define a CPUKernels class.

Reply:
Okay!